src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/PlacementApplicationServiceTest.kt

```diff
@@ -244,8 +248,9 @@ class PlacementApplicationServiceTest {
 
       every { userService.getDeliusUserNameForRequest() } returns "theUsername"
       every { cas1PlacementApplicationDomainEventService.placementApplicationSubmitted(any(), any()) } returns Unit
-      every { cas1PlacementApplicationEmailService.placementApplicationSubmitted(any()) } returns Unit
-      every { cas1PlacementApplicationEmailService.placementApplicationAllocated(any()) } returns Unit
+      every { cas1PlacementApplicationEmailService.placementApplicationSubmitted(any()) } just Runs
+      every { cas1PlacementApplicationEmailService.placementApplicationAllocated(any()) } just Runs
+      every { cas1PlacementApplicationDomainEventService.placementApplicationAllocated(any(), user) } just Runs
```

Comment:
Should we verify that the domain event has been created somewhere?

Reply:
I've updated the tests to include this.